src/main/java/Duke.java

```diff
@@ -1,8 +1,51 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = "Hello! I'm Yube \n"
-                + "What can I do for you? \n" +
-                "Bye. Hope to see you again soon!";
-        System.out.println(logo);
+    private static final String LINE = "___________________________________________________________\n";
+
+    public static void main(String[] args) throws IOException {
+        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+        boolean hasEnded = false;
+        String botName = "Yube";
+
+        greet(botName);
+        while (!(hasEnded)) {
+            String input = reader.readLine();
+            if (input.equals("bye")) {
+                bye();
+                hasEnded = true;
+            } else {
+                repeatFunction(input);
+            }
+        }
+    }
+
+    /**
+     * Displays a greeting message
```

Comment:
I think the sentence should end with a fullstop. I noticed the same issue in several other places too.